infra/stage/locals.tf

```diff
@@ -0,0 +1,3 @@
+locals {
+  iam_admin_users = ["rsalmond", "Ian_Edington_Laptop"]
```

Comment:
I wonder if this could be a security concern when more people join. It shares publicly who to target with social engineering.